.gitignore

```diff
@@ -106,6 +106,7 @@ ENV/
 
 # VSCode
 settings.json
+*.code-workspace
```

Comment:
Is that line necessary? For which IDE?

What do you think @bilderbuchi?

Reply:
It's for VS Code (see the comment a bit above). I think it's for multi-project workspace setups (i.e. having pymeasure and another package in one editing environment).
I think it's OK.

Reply:
Ok, thanks.